.travis.yml

```diff
@@ -4,11 +4,36 @@ os:
   - linux
 python:
   - '3.6'
+sudo: enabled
+addons:
+  apt:
+    packages:
+    - mpich
+    - libmpich-dev
+    - libglib2.0-0
+    - libgl1-mesa-glx
```

Comment:
we don't need this. This is to install spikyng-circus. We only need klusta, ml_ms4_alg, and tridesclous for the examples